components/cloak-visual.coffee

```diff
@@ -65,9 +65,13 @@ export default
 	# Render a Visual instance
 	render: (create, { props, injections, data, children, scopedSlots }) ->
 
+		# if the file is a PNG, then don't show a placeholder
+		img = props.image?[0]
+		isPng = img?.path.match(/.(png)$/i) || img?.mimeType == 'image/png'
```

Comment:
@mattaebersold `.mimeType` sounds like it's Craft specific, so maybe this should go one up the tree in `craft-visual.coffee`?  

Contentful has a different property called `contentType`, so the equivalent (untested) line you could put in `contentful-visual.coffee` would be:

`isPng = img?.path.match(/.(png)$/i) || img?.contentType == 'image/png'`

Reply:
I pushed up a new commit with these changes:

- Moves logic later so if no image value was supplied, we still show placeholder
- Uses only the image suffix as a signal for transparency support
- Adds a `autoNoPlaceholder` prop so this behavior can be disabled, though it defaults to true because I think that's unlikely

If we want to also trigger on mime-type, we'd add some additional code to `craft-visual` and `contentful-visual` respectively, since, like @jonjahr pointed out, mimetype is stored differently there.  I'm cool shipping w/o though, I think this would cover most use cases.